crates/solver/src/liquidity.rs

```diff
@@ -75,13 +78,49 @@ pub enum Exchange {
     ZeroEx,
 }
 
+#[derive(Debug, Clone)]
+#[cfg_attr(test, derive(Derivative))]
+#[cfg_attr(test, derivative(PartialEq))]
+pub enum LimitOrderUid {
+    OrderUid(OrderUid),
+    ZeroEx(String),
```

Comment:
ubernit: Instead of making this 0x specific, we can:
```suggestion
    Other(String),
```

This way, adding additional limit orders doesn't need new variants.

Reply:
But it does at a cost of not knowing what `Other` could be (I need to look further in the code to see where `Other` is used).

Reply:
Good point. I think its fine as-is, and we can evaluate if/when we add additional liquidity types.